types/lolex/index.d.ts

```diff
@@ -110,26 +112,31 @@ export interface LolexClock<TTimerId extends TimerId> {
 
     /**
      * Clears a timer, as long as it was created using setImmediate.
-     * 
+     *
      * @param id   Timer ID or object.
      */
     clearImmediate(id: TTimerId): void;
 
+    /**
+     * Simulates process.nextTick();
+     */
+    nextTick(callback: () => any): void;
```

Comment:
I think it's more accurate to say the callback returns `void`, since Lolex doesn't use the returned value.

Reply:
fixed